pkg/dynamic/client.go

```diff
@@ -29,15 +29,20 @@ import (
 )
 
 const (
-	crdGroup          = "apiextensions.k8s.io"
-	crdVersion        = "v1beta1"
-	crdKind           = "CustomResourceDefinition"
-	crdKinds          = "customresourcedefinitions"
-	sourcesLabelKey   = "duck.knative.dev/source"
-	sourcesLabelValue = "true"
-	sourceListGroup   = "client.knative.dev"
-	sourceListVersion = "v1alpha1"
-	sourceListKind    = "SourceList"
+	crdGroup           = "apiextensions.k8s.io"
+	crdVersion         = "v1beta1"
+	crdKind            = "CustomResourceDefinition"
+	crdKinds           = "customresourcedefinitions"
+	sourcesLabelKey    = "duck.knative.dev/source"
+	sourcesLabelValue  = "true"
+	sourceListGroup    = "client.knative.dev"
+	sourceListVersion  = "v1alpha1"
+	sourceListKind     = "SourceList"
+	channelLabelKey    = "messaging.knative.dev/subscribable"
+	channelLabelValue  = "true"
+	channelListGroup   = "messaging.knative.dev"
+	channelListVersion = "v1beta1"
+	channelListKind    = "ChannelList"
```

Comment:
Wherever applicable, we should refer these values from vendored eventing code, good candidate for a subsequent PR.